admin/load.php

```diff
@@ -138,18 +138,13 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 			<?php if ( perflab_can_load_module( $module_slug ) ) { ?>
 				<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" aria-describedby="<?php echo esc_attr( "{$base_id}_description" ); ?>" value="1"<?php checked( $enabled ); ?>>
 				<?php
-				if ( $module_data['experimental'] ) {
 					printf(
 						/* translators: %s: module name */
-						__( 'Enable %s <strong>(experimental)</strong>', 'performance-lab' ),
-						esc_html( $module_data['name'] )
-					);
-				} else {
-					printf(
-						/* translators: %s: module name */
-						__( 'Enable %s', 'performance-lab' ),
+						esc_html__( 'Enable %s', 'performance-lab' ),
 						esc_html( $module_data['name'] )
 					);
+				if ( $module_data['experimental'] ) {
+					echo ' <strong>' . esc_html__( '(experimental)', 'performance-lab' ) . '</strong>';
 				}
```

Comment:
This fix is not right, translation strings should not be concatenated. Please use the string as it was before.

What we need to change is how it is being escaped. We cannot use `esc_html()` here because it contains HTML, so we need to escape it so that only that specific HTML is allowed. Please use `wp_kses( $translated_string, array( 'strong' => array() ) )`. This way the string is safe, the only allowed HTML will be the `<strong>` tag.

Reply:
Thanks for pointing it out. I have address this in https://github.com/WordPress/performance/pull/695/commits/071ffacf0181bf7c3370d3d11a666e5a1ec54a87